tools/swift-format/Sources/Rules/DeclSyntax+Comments.swift

```diff
@@ -19,7 +19,31 @@ extension DeclSyntax {
         if hasSeenFirstLineComment {
           break gatherComments
         }
-        return text
+        let blockComment = text.components(separatedBy: "\n")
+        // Removes the marks of the block comment.
+        var isTheFirstLine = true
+        let blockCommentWithoutMarks = blockComment.map { (line: String) -> String in
+          // Only the first line of the block comment start with '/**'
+          let markToRemove = isTheFirstLine ? "/**" : "* "
+          if line.trimmingCharacters(in: .whitespaces).starts(with: markToRemove) {
+            let numCharsToRemove = isTheFirstLine ? markToRemove.count : markToRemove.count - 1
+            isTheFirstLine = false
+            return line.hasSuffix("*/") ?
+            String(line.trimmingCharacters(in: .whitespaces).dropFirst(numCharsToRemove).dropLast(2)) :
```

Comment:
Extract the common subexpression out here, so that you have:

```swift
let trimmedLine = line.trimmingCharacters(in: .whitespaces).dropFirst(numCharsToRemove)
return String(line.hasSuffix("*/") ? trimmedLine.dropLast(2) : trimmedLine)
```